products.html

```diff
@@ -0,0 +1,335 @@
+                                <div class="card-v-image card-prod-image">
+                                    <img class="img-responsive" src="images/optimumNutrition.png" alt="optimumNutrition"/>
+                                </div>
+                                <div class="card-v-content">
+                                    <p class="txt-bold p-md">Optimum Nutrition</h2>
+                                    <p class="p-md">Gold Standard Whey Protein</h2>
+                                    <p class="card-price p-lg txt-bold">$37.99 <span class="txt-linethrough"> $41.99</span></p>
+                                    <p class="card-price p-sm color-pri txt-bold">Free Delivery!! </p>
+                                </div>
+                            </div>
+                            <div class="card-vertical card-prod-mini">
+                                <div class="card-v-header flex-row">
+                                    <span class="badge badge-orange">New Arrival</span>
+                                    <div class="prod-header-sec2 flex-row">
+                                        <button class="primary-btn-icon p-lg"><i class="far fa-heart"></i></button>
+                                        <button class="primary-btn-icon p-lg"><i class="fas fa-cart-plus"></i></button>
+                                    </div>
+                                </div>
+                                <div class="card-v-image card-prod-image">
+                                    <img class="img-responsive" src="images/optimumNutrition2.png" alt="optimumNutrition"/>
+                                </div>
+                                <div class="card-v-content">
+                                    <p class="txt-bold p-md">Optimum Nutrition</h2>
+                                    <p class="p-md">Serious Mass Weight Gainer</h2>
+                                    <p class="card-price p-lg txt-bold">$29.99 <span class="txt-linethrough"> $36.99</span></p>
+                                    <p class="card-price p-sm color-pri txt-bold">Delivery by tomorrow!!</p>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </main>
+
+        <footer class="footer flex-column">
+            <div class="footer-content flex-row ">
+                <div class="footer-column flex-column">
+                    <span class="footer-headings txt-bold p-lg">Get to know us</span>
+                    <button class="primary-btn-link p-md txt-bold">Contact</button>
+                    <button class="primary-btn-link p-md txt-bold">About Us</button>
+                    <button class="primary-btn-link p-md txt-bold">Careers</button>
+                    <button class="primary-btn-link p-md txt-bold">Press Releases</button>
+                    <button class="primary-btn-link p-md txt-bold">Stories</button>
```

Comment:
you should prefer the links instead btns

Reply:
changed done and pushed.  Thanks for the review Mahendra